### ISSUE TITLE
Question: difference to sklearn.pipeline

### ISSUE THREAD

**Issue by hehoffmann15** [KIT-IAI/pyWATTS#38]
I found your project very interesting. I am working with time series data and right now I am using sklearn pipelines for automation of tasks. So my question is: what is the advantage in using your tool compared to the sklearn pipelines? What are the differences and what is your major improvement that makes your tool so important?

**Comment by benHeid** [comment]
Hey, @hehoffmann15 , 
sklearn allows only linear execution of modules, where neither parallel nor conditional execution is possible. In contrast, pyWATTS allows non-sequential pipelines, e.g. the branching and merging of paths in the pipeline is possible. In the future, this should also allow us to execute steps of the pipeline in parallel. Additionally, we have a condition mechanism that decides which module is executed depending on the characteristics of the applied data.  Moreover, we integrate third party modules by providing wrappers (currently for sklearn, PyTorch and Keras).
